st3/sublime_lib/pure_resource_path.py

```diff
@@ -103,13 +114,20 @@ def match(self, pattern):
         return matcher.match(str(self))
 
     def joinpath(self, *other):
+        """Combine this path with all of the given strings."""
         return self.__class__(self, *other)
 
     def with_name(self, name):
+        """Return a new path with the name changed.
+
+        :raise ValueError: if the original path doesn’t have a name."""
         if not self.name:
             raise ValueError("{!r} has an empty name".format(self))
 
         return self.parent / name
 
     def with_suffix(self, suffix):
+        """Return a new path with the suffix changed. If the original path
```

Comment:
Should begin a new paragraph after the first sentence (for the short description).

Reply:
Fixed.